app/allocation/controllers/view.js

```diff
@@ -0,0 +1,18 @@
+const presenters = require('../../../common/presenters')
+
+module.exports = function view(req, res) {
+  const { allocation } = res.locals
+  const {
+    allocation: { moves_count: movesCount, moves },
+  } = res.locals
```

Comment:
It looks like `res.locals` is being destructured twice, could them be combined?

Reply:
As far as I am aware, you can't destructure an object at several levels in one pass.

Reply:
I have replaced the deep destructuring now.